website/content/en/preview/tasks/scheduling.md

```diff
@@ -253,6 +254,34 @@ The three supported `topologyKey` values that Karpenter supports are:
 
 See [Pod Topology Spread Constraints](https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/) for details.
 
+## Pod affinity/anti-affinity
+
+By using the `podAffinity` and `podAntiAffinity` configuration on a pod spec, you can ask inform the provisioner of your desire for pods to schedule together or apart with respect to different topology domains. For example:
+
+```yaml
+spec:
+  affinity:
+    podAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+          - key: system
+            operator: In
+            values:
+            - backend
+        topologyKey: topology.kubernetes.io/zone
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchLabels:
+            app: inflate
+        topologyKey: kubernetes.io/hostname
+```
+
+The above pod affinity rule would cause the pod to only schedule in zones where a pod with the label `system=backend` is already running.  
+
+The anti-affinity rule would cause it to avoid running on any node with a pod labeled `app=inflate`.  If this anti-affinity term was on a deployment pod spec along with a matching `app=inflate` label, it would prevent more than one pod from the deployment from running on any single node. 
```

Comment:
Can you add a link to the pod affinity kubernetes docs too? like we have for the Pod Topology Spread section